system/settings/settings.xml

```diff
@@ -3791,6 +3791,18 @@
             <multiselect>true</multiselect>
           </control>
         </setting>
+        <setting id="locale.fontinstall" type="integer" label="2108" help="2109">
+          <level>1</level>
+          <default>1</default> <!-- InstallWay::NO_ASK -->
```

Comment:
You can use `InstallMethod` instead. "Way" means pretty much anything but "method" means that a particular procedure is used.

Reply:
Have changed to "method", also used it as this on the used strings.po.